libs/application/templates/parental-leave/src/fields/InReviewSteps/ReviewSection/index.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { FC } from 'react'
+import cn from 'classnames'
+
+import { useLocale } from '@island.is/localization'
+import { Application, formatText } from '@island.is/application/core'
+import { Box, Icon, Tag, Text } from '@island.is/island-ui/core'
+
+import * as styles from './ReviewSection.treat'
+
+export type reviewSectionState =
+  | 'In progress'
+  | 'Requires action'
+  | 'Complete'
+  | undefined
```

Comment:
should this be translatable or is this just used as a key? If so, then this should probably be an enum

Reply:
Yeah it's used just as a key, switch to an enum now (great suggestion thanks!).